plugins/vendor/lib/model/VendorIntegration.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+
+/**
+ * Skeleton subclass for representing a row from the 'vendor_integration' table.
+ *
+ * 
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements.  This class will only be generated as
+ * long as it does not already exist in the output directory.
+ *
+ * @package plugins.vendor
+ * @subpackage model
+ */
+class VendorIntegration extends BaseVendorIntegration {
+
+	public function setAccessToken ($v)	{ $this->putInCustomData ( "accessToken" , $v);	}

```

Comment:
Move strings such as "accessToken" & "refreshToken" & etc ... to be const on the class